src/SymOp.jl

```diff
@@ -19,21 +19,19 @@
 # Tolerance to consider two atomic positions as equal (in relative coordinates)
 const SYMMETRY_TOLERANCE = 1e-5
 
-# Represents a symmetry (W,w)
 struct SymOp{T <: Real}
     W::Mat3{Int}
     w::Vec3{T}
+
+    S::Mat3{Int}
+    τ::Vec3{T}
     function SymOp(W, w)
         w = mod.(w, 1)
-        new{eltype(w)}(W, w)
+        S = symop.W'
+        τ = -symop.W \symop.w
+        new{eltype(τ)}(W, w, S, τ)
     end
```

Comment:
why is this actually needed in-struct?

Reply:
it's not but for simple constructors it's simpler to have it there no?

Reply:
I prefer the out-of-struct constructions because it is nicer to add multiple dispatch etc. and we basically only have those at the moment.

Reply:
In this particular case it's better like this because having an inner constructor removes the default constructor (which takes as many inputs as in the struct), so here the invariants are guaranteed by construction

Reply:
ok